docs/siem-apis.asciidoc

```diff
@@ -8,8 +8,9 @@ You can use these APIs to interface with {elastic-sec} features:
 NOTE: Console supports sending requests to {kib} APIs. Prepend any {kib} API endpoint with `kbn:` and send the request via Console. For example:
 `GET kbn:/api/index_management/indices`
 
-* <<rule-api-overview>>: Manage detection rules and alerts
-* <<exceptions-api-overview>>: Create and manage rule exceptions
+* <<rule-api-overview>>: Manage detection rules, rule exceptions for individual rules, and alerts
+* <<exceptions-api-overview>>: Create and manage shared rule exceptions 
+//Will finalize descriptions once get confirmation from Yara about exception API endpoints
```

Comment:
What's this comment here? How can I help?

Reply:
No need! Removed this and tweaked the description in [585ed2c](https://github.com/elastic/security-docs/pull/4021/commits/585ed2cb75a83d00203852718b0a7d5fb610f4f6).